hydradx/spec/Swap.ipynb

```diff
@@ -91,8 +91,10 @@
     "$$\n",
     "\\begin{align}\n",
     "\\Delta Q_i &= Q_i\\frac{- \\Delta R_i}{R_i + \\Delta R_i}\\\\\n",
-    "\\Delta Q_j' &= -\\Delta Q_i (1 - f_P)\\\\\n",
-    "\\Delta R_j' &= R_j\\frac{- \\Delta Q_j'}{Q_j + \\Delta Q_j'}\\\\\n",
+    "\\Delta Q_j^t' &= -\\Delta Q_i (1 - f_P)\\\\\n",
```

Comment:
We should not be adding the LRNA minting feature to the spec, we added it to the implementation for testing, not because we have decided the Omnipool will be changed (yet).

Reply:
I'm marking this as resolved because you added the commit to revert this change